modules/web-cli/assets/js/core/commands.js

```diff
@@ -19,6 +19,23 @@ export default class Commands extends CommandsBackwardsCompatibility {
 		this.currentTrace = [];
 		this.commands = {};
 		this.components = {};
+
+		Object.defineProperty( this, 'classes', {
+			get() {
+				elementorCommon.helpers.softDeprecated(
+					'$e.commands.classes',
+					'3.7.0',
+					'$e.commands.getCommandClass(), $e.commandsInternal.getCommandClass(), $e.data.getCommandClass(), $e.routes.getCommandClass() according to the requested command infra-structure,'
+				);
+
+				return {
+					... $e.commands.commands,
+					... $e.commandsInternal.commands,
+					... $e.data.commands,
+					... $e.routes.commands,
```

Comment:
Why return all? Each one should return its own commands.

Reply:
`$e.commands.classes` in BC included all the commands.

Reply:
I see, but why?

Reply:
Explained in private.